src/model/CoilCoolingDXTwoSpeed.hpp

```diff
@@ -85,6 +85,10 @@ namespace model {
    *  Optional == false means "Autocalculate" */
     boost::optional<double> ratedHighSpeedAirFlowRate() const;
 
+    double ratedHighSpeedEvaporatorFanPowerPerVolumeFlowRate2017() const;
+
+    double ratedHighSpeedEvaporatorFanPowerPerVolumeFlowRate2023() const;
```

Comment:
Example for the new Fan power per vol flow rate 2017 / 2023. Added to model here.